docs/tutorials/rich-html-artifacts/fixtures/revops-launch-risk/artifact.html

```
@@ -0,0 +1,158 @@
+
+        <div class="owner-load" aria-label="Owner load">
+          <div class="owner"><strong>Finance</strong><div class="bar"><span style="width: 78%"></span></div><p>Approval path is the launch gate.</p></div>
+          <div class="owner"><strong>Support</strong><div class="bar"><span style="width: 44%"></span></div><p>Visual refresh is bounded.</p></div>
+          <div class="owner"><strong>Sales</strong><div class="bar"><span style="width: 63%"></span></div><p>Buyer narrative still needs proof.</p></div>
+        </div>
+      </section>
+
+      <section>
+        <h2>Risk Tuning</h2>
+        <div class="controls">
+          <label>Pricing severity <input id="pricing" type="range" min="0" max="100" value="82"></label>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)